course6/toyapp/CMakeLists.txt

```diff
@@ -1,10 +1,11 @@
 project(toyapp)
 cmake_minimum_required(VERSION 3.0)
 
+link_directories(C:/lib/SDL2/lib)
```

Comment:
I've found an elegant solution for this:
```
if (SDL_ROOT)
   include_directories(${SDL_ROOT}/include)
   link_directories(${SDL_ROOT}/lib)
endif()
```
This way you only have to set ```SDL_ROOT ``` variable in your project's config, which you can do in the CMake GUI, or by setting it in command-line, and you don't need to hardcode the path in your CMakeLists.txt